### ISSUE TITLE
SketchyBar does not show up

### ISSUE THREAD

**Issue by Xorger** [FelixKratz/SketchyBar#618]
I'm attempting to use SketchyBar on my Intel Core i9 iMac 27" from 2020, However the bar just refuses to render (I've made sure it's not just behind all the windows). I'm using the default config from the install guide, and have tried the steps outlined in #457 with no luck. Apologies if this issue is answered somewhere else. Please feel free to ask for more info.

**Comment by zeidlos** [comment]
Please just run `sketchybar` in your terminal and post its output here.

You might get an error stating 
```sketchybar: 'System Settings' -> 'Desktop & Dock' -> 'Displays have separate spaces' needs to be enabled for sketchybar to work properly.``` 

Then you know what to do. :)

**Comment by Xorger** [comment]
Nope, sorry, SketchyBar just never exits and never gives any errors/output.

**Comment by winiciusallan** [comment]
same here. I ran `sketchybar` from command line and nothing happens. no error log nor some output.

```bash
> sketchybar
...
```

just prompting forever. any ideas? i've already tried to reboot the pc and reinstall the sketchybar

**Comment by charlesnicholson** [comment]
This wasn't working for me either when running `sketchybar` from my terminal. 

But, when I enabled `sketchybar` as a service through brew and then logged out + back in, it showed up!

**Comment by 9SMTM6** [comment]
I thought it wasn't showing up, then I realized that it worked, but my autotiling aerospace of course doesn't respect it.

**Comment by Xorger** [comment]
I just came back to this, it seems to be working now, however it's behind all the windows. I probably need to change my yabai conf.
I don't know if something changed in the few months since i made this issue, so closing for now
